lib/plugins/aws/package/compile/events/apiGateway/lib/stage.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
       // --------------------------------------------------------------------------------
 
       // create log-specific resources
-      if (logs) {
+      if (logs === true || (logs && logs.accessLogging)) {
```

Comment:
I think by default `logs.accessLogging` is true (?) and if that's the case, then this will exclude users which use object configuration but do not set `logs.accessLogging` explicitly

I guess the most solid in such case would be:

```
if (logs && (logs.accessLogging == null || logs.accessLogging)) {
```

Reply:
That's a good point, I've missed that. Adjusted + added tests cases to ensure it's not going to break for other configurations.